index.html

```diff
@@ -4292,6 +4306,489 @@ <h3>Proof Formats</h3>
+
+        <p>
+The algorithm in this section can be used to determine if a provided document is
+conformant to the data model provided in this specification. This algorithm
+takes a document (object <var>inputDocument</var>) and produces an object
+(<var>dataModelConformanceChecked</var>) that contains the following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The data model verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>dataModelConformanceChecked</var> to an empty object. Initialize the following
+fields in <var>dataModelConformanceChecked</var>: <var>status</var> to `false`,
+<var>warnings</var> to an empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Ensure that the `@context` property of <var>document</var> conforms to the
+statements in <a href="#contexts"></a>. If not, return
+<var>dataModelConformanceChecked</var>, which will contain a false verification status.
+          </li>
+          <li>
+Starting at the top level, recursively process each object <var>obj</var> in
+<var>document</var>, setting <a href="#MALFORMED_VALUE_ERROR">
+MALFORMED_VALUE_ERROR</a> errors in
+<var>dataModelConformanceChecked</var>.<var>errors</var> if errors are found, by
+performing the following checks:
+            <ol class="algorithm">
+              <li>
+If present, ensure that the `id` property conforms to the "MUST" statements in
+Section <a href="#identifiers"></a>.
+              </li>
+              <li>
+If present, ensure that the `type` property conforms to the "MUST" statements in
+Section <a href="#types"></a>.
+              </li>
+              <li>
+If present, ensure that the `name` and/or `description` properties conform to
+the "MUST" statements in Section <a href="#names-and-descriptions"></a>.
+              </li>
+              <li>
+If the `type` property is present and contains the
+`VerifiablePresentation` type, ensure that the object conforms to the
+"MUST" statements in Section <a href="#presentations-0"></a>.
+              </li>
+              <li>
+If the `type` property is present and contains the
+`VerifiableCredential` type, perform the following checks:
+                <ol class="algorithm">
+                  <li>
+Ensure that the `credentialSubject` property conforms to the
+"MUST" statements in Section <a href="#credential-subject"></a>.
+                  </li>
+                  <li>
+Ensure that the `issuer` property conforms to the
+"MUST" statements in Section <a href="#issuer"></a>.
```

Comment:
If we keep this section, what value is this providing over the existing normative requirements of conforming documents?

Is this section of MUSTs really just reminding us which fields are "required" and "optional" and to read the section that defines them?

Since `proof` was removed, but is still optional, does this section still make sense?

@jyasskin help me understand why you feel the repetition is helpful here, since we end up needing to read all the sections anyway.

Reply:
Per https://github.com/w3c/vc-data-model/pull/1338#discussion_r1414492504, I don't insist on repeating the list of properties.